### PR TITLE
Create initial MRTCore release pipeline file

### PR DESCRIPTION
**Pull request by huichen123**

### CODE REVIEW

File: dev/MRTCore/mrtcorerelease.yml [hunk 1]
```diff
@@ -0,0 +1,650 @@
...
+# Build a Universal Windows Platform project using Visual Studio.
+# Add steps that test and distribute an app, save build artifacts, and more:
+# https://aka.ms/yaml
+
+name: $(BuildDefinitionName)-$(date:yyMM).$(date:dd)$(rev:rrr)
+
+#trigger:
+#- master
+
+jobs:
```
Comment (EHO-makai) [inline dev/MRTCore/mrtcorerelease.yml:11]: This file is quite a bit longer than WinUI2's MUX-Release.yml.  Is this duplicating stuff from the CI build?  Can this accept the output of another YML and just do sign and publish?  This is essentially what it looks like the WinUI2 release yml does.

Comment (huichen123) [reply dev/MRTCore/mrtcorerelease.yml:11]: we can make improvement to the files later. for now, I'd like to get things started. Without a working pipeline, I'd rather avoid the unnecessary content change for now.

Comment (axelandrejs) [reply dev/MRTCore/mrtcorerelease.yml:11]: Agreed, we should get it working ASAP then make a follow-up change to prune if needed.

**Review by axelandrejs — APPROVED** [review]

File: dev/MRTCore/mrtcorerelease.yml (hunk 1 above)
Comment (huichen123) [reply dev/MRTCore/mrtcorerelease.yml:11]: @EHO-makai  should i proceed or wait?

**Pull request closed without merging**